### PR TITLE
Test/add tranform integr tests + hide email

### PR DESCRIPTION
**Pull request by cpommier**

### CODE REVIEW

File: etl/transform/elasticsearch.py [hunk 1]
```diff
@@ -132,15 +132,19 @@ def validate_documents(document_tuples, validation_schemas, logger):
...
+    Consumes an iterable of document tuples and clean email
     """
     logger.debug("Saving documents to json files...")
 
     json_list = list()
     document_count = 0
     for document_header, document in documents_tuples:
+        if "contact" in document:
+            old_mail = document["contact"]["email"]
+            safe_mail = old_mail.replace('@', '_')
```
Comment (raf64flo) [inline etl/transform/elasticsearch.py:146]: Is `email` mandatory? What if None?

File: tests/transform/integration/test_es_integration.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    
+    def setUp(self):
+        current_path = os.path.dirname(__file__)
+        root_dir = os.path.normpath(os.path.join(current_path, "../../../"))
+        result = subprocess.run([root_dir + "/main.py", "trans", "es", root_dir + "/tests/transform/integration/fixtures/VIB.json", "--data-dir", root_dir + "/tests/transform/integration/fixtures/brapi_pheno_source"])
+
+    #("pipenv run ./main.py trans es --data-dir sandbox/ sources/VIB.json sources/NIB.json")
+
+    def test_toto(self):
+        self.assertEqual(1,1)
```
Comment (raf64flo) [inline tests/transform/integration/test_es_integration.py:16]: Obvious, but useless IMHO. :smile:

File: tests/transform/integration/test_es_integration.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    
+    def setUp(self):
+        current_path = os.path.dirname(__file__)
+        root_dir = os.path.normpath(os.path.join(current_path, "../../../"))
+        result = subprocess.run([root_dir + "/main.py", "trans", "es", root_dir + "/tests/transform/integration/fixtures/VIB.json", "--data-dir", root_dir + "/tests/transform/integration/fixtures/brapi_pheno_source"])
+
+    #("pipenv run ./main.py trans es --data-dir sandbox/ sources/VIB.json sources/NIB.json")
+
+    def test_toto(self):
```
Comment (raf64flo) [inline tests/transform/integration/test_es_integration.py:15]: To rename to reflect the aim of the test.

File: tests/transform/integration/test_es_integration.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import subprocess
+import os
+
+class MyTestCase(unittest.TestCase):
+
+    
+    def setUp(self):
+        current_path = os.path.dirname(__file__)
+        root_dir = os.path.normpath(os.path.join(current_path, "../../../"))
+        result = subprocess.run([root_dir + "/main.py", "trans", "es", root_dir + "/tests/transform/integration/fixtures/VIB.json", "--data-dir", root_dir + "/tests/transform/integration/fixtures/brapi_pheno_source"])
```
Comment (raf64flo) [inline tests/transform/integration/test_es_integration.py:11]: The generated files should be `diff` with the fixtures for the test to be relevant, right?

**Review by raf64flo — CHANGES_REQUESTED** [review]
1. Test to be completed.
2. Check of `email != None` to be done.

File: etl/transform/elasticsearch.py (hunk 1 above)
Comment (cpommier) [reply etl/transform/elasticsearch.py]: right

Comment (cpommier) [reply etl/transform/elasticsearch.py]: Done, a little bit more complicated than expected

File: tests/transform/integration/test_es_integration.py (hunk 4 above)
Comment (cpommier) [reply tests/transform/integration/test_es_integration.py]: Yes, done in each test_all

**Pull request merged**